File: 2068-check-whether-two-strings-are-almost-equivalent/2068-check-whether-two-strings-are-almost-equivalent.c
bool checkAlmostEquivalent(char * word1, char * word2){
    int hash[26] = {0};
    
    for(int i = 0; word1[i]; i++)
        hash[word1[i] - 'a']++;
    for(int i = 0; word1[i]; i++)
        hash[word2[i] - 'a']--;
    for(int i = 0; i < 26; i++){
        if(abs(hash[i]) > 3)
            return false;
    }
    return true;
}

File: 2068-check-whether-two-strings-are-almost-equivalent/2068-check-whether-two-strings-are-almost-equivalent.java
class Solution {
    public boolean checkAlmostEquivalent(String word1, String word2) {
        int[] hash = new int[26];
    
        for(int i = 0; i < word1.length(); i++)
            hash[word1.charAt(i) - 'a']++;
        for(int i = 0; i < word2.length(); i++)
            hash[word2.charAt(i) - 'a']--;
        for(int i = 0; i < 26; i++){
            if(Math.abs(hash[i]) > 3)
                return false;
        }
        return true;
    }
}